pkgs/tools/networking/dublin-traceroute/default.nix

```
@@ -0,0 +1,45 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, pkg-config
+, jsoncpp
+, libtins
+, libpcap
+, openssl
+, nixosTests
+}:
+
+stdenv.mkDerivation {
+  pname = "dublin-traceroute";
+  version = "unstable-2023-04-12";
+
+  src = fetchFromGitHub {
+    owner = "insomniacslk";
+    repo = "dublin-traceroute";
+    rev = "2fb78ea05596dfdf8f7764b497eb8d3a812cb695";
+    sha256 = "sha256-E1HYMd0wDTfAZ0TamQFazh8CPhMa2lNIbF4aEBf5qhk=";
+  };
+
+  nativeBuildInputs = [ cmake pkg-config ];
+
+  buildInputs = [ jsoncpp libtins libpcap openssl ];
+
+  outputs = [
+    "out"
+    "lib"
+    "dev"
+  ];
+
+  meta = with lib; {
+    description = "Dublin Traceroute is a NAT-aware multipath traceroute tool.";
+    homepage = "https://dublin-traceroute.net/";
+    license = licenses.bsd2;
+    maintainers = with maintainers; [ baloo ];
+    platforms = platforms.unix;
```

I believe the description needs to be in another "format": https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#meta-attributes

Also added `mainProgram`

```suggestion
    description = "NAT-aware multipath traceroute tool";
    homepage = "https://dublin-traceroute.net/";
    license = licenses.bsd2;
    maintainers = with maintainers; [ baloo ];
    platforms = platforms.unix;
    mainProgram = "dublin-traceroute";
```